.github/workflows/contract-test.yml

```diff
@@ -58,7 +62,7 @@ jobs:
       run: npm install
     - name: Run tests
       working-directory: contracts
-      run: npm run test:gw_testnet_v1
+      run: npm run test:${{ matrix.network }}
```

Comment:
Should we use different private keys for different networks?

Reply:
Not necessary since they are different layer2 networks, so conflicts will not happen.

Reply:
Maybe then we should change the `GODWOKEN_TESTNET_PRIVATE_KEY` secrets' name to something else like `GODWOKEN_TEST_PRIVATE_KEY` to better suite the case? 

Do you thinks it's necessary? Neither way is urgent tho.

Reply:
Yes, please update the names of the secrets in another PR when you have a moment.